app/javascript/stylesheets/login.scss

```diff
@@ -0,0 +1,75 @@
+.scene-1-login {
+  position: absolute;
+  display: flex;
+  top: 8rem;
+  left: 50%;
+  transform: translate(-50%, 0%);
+  z-index: -1;
+}
+
+.login-content {
+  padding: 24px;
+  position: absolute;
+  display: block;
+  left: 50%;
+  transform: translate(-50%, 0%);
+  top: 23rem;
+  width: 100%;
+  text-align: center;
+
+  .login-title {
+    font-weight: 600;
+    font-size: 21px;
+    display: flex;
+    text-align: center;
+    color: #252636;
+    justify-content: center;
+  }
+
+  form {
```

Comment:
**issue (blocking)**: avoid adding stiles diretly to html elements, use a class instead, otherwise it will be much harder to override styles in edge cases